gravitee-am-service/src/test/java/io/gravitee/am/service/CertificateServiceTest.java

```diff
@@ -195,76 +245,155 @@ public void shouldDelete_certificateWithClients() {
-                        "      \"title\": \"PKCS#12 file\",\n" +
-                        "      \"description\": \"PKCS file\",\n" +
-                        "      \"type\" : \"string\",\n" +
-                        "      \"widget\" : \"file\"\n" +
-                        "    },\n" +
-                        "    \"storepass\" : {\n" +
-                        "      \"title\": \"Keystore password\",\n" +
-                        "      \"description\": \"The password which is used to protect the integrity of the keystore.\",\n" +
-                        "      \"type\" : \"string\"\n" +
-                        "    },\n" +
-                        "    \"alias\" : {\n" +
-                        "      \"title\": \"Key alias\",\n" +
-                        "      \"description\": \"Alias which identify the keystore entry.\",\n" +
-                        "      \"type\" : \"string\"\n" +
-                        "    },\n" +
-                        "    \"keypass\" : {\n" +
-                        "      \"title\": \"Key password\",\n" +
-                        "      \"description\": \"The password used to protect the private key of the generated key pair.\",\n" +
-                        "      \"type\" : \"string\"\n" +
-                        "    }\n" +
-                        "  },\n" +
-                        "  \"required\": [\n" +
-                        "    \"content\",\n" +
-                        "    \"storepass\",\n" +
-                        "    \"alias\",\n" +
-                        "    \"keypass\"\n" +
-                        "  ]\n" +
-                        "}"));
-        TestObserver<Certificate> testObserver = certificateService.create("my-domain").test();
+        when(certificatePluginService.getSchema(DEFAULT_CERTIFICATE_PLUGIN))
+                .thenReturn(Maybe.just(CERTIFICATE_SCHEMA_DEFINITION));
+
+        TestObserver<Certificate> testObserver = certificateService.rotate(DOMAIN, mock(User.class)).test();
         testObserver.awaitTerminalEvent();
-//        testObserver.assertComplete();
-        return testObserver;
+
+        verify(certificateRepository).create(argThat(cert -> cert.isSystem()
+                && cert.getDomain().equals(DOMAIN)
+                && cert.getName().matches("Default\\s[\\d]+")
+                && cert.getConfiguration().contains("[\"sig\"]")
+                && cert.getConfiguration().contains("RS256")
+        ));
+    }
+
+    @Test
+    public void shouldRotate_defaultCertificate_Rsa_firstDefault() throws Exception {
+        final var now = LocalDateTime.now();
+
+        final Certificate certCustom = new Certificate();
+        certCustom.setSystem(false);
+        certCustom.setDomain(DOMAIN);
+        certCustom.setName("Cert-4");
+        certCustom.setConfiguration("{\"content\":\"03c14ae4-5f9b-4636-814a-e45f9bb636c4.p12\",\"storepass\":\"gravitee\",\"alias\":\"default\",\"keypass\":\"gravitee\"}");
```

Comment:
Same here. If the certificate configuration ever needs to change, it will be difficult updating these.